targets/csharp/source/PlayFabSDK/source/Uunit/tests/HttpTests.cs

```diff
@@ -81,7 +81,6 @@ public void TestPluginWithoutPolly_OnSuccess_200Response(UUnitTestContext testCo
         [UUnitTest]
         public void TestPluginWithPolly_Success(UUnitTestContext testContext)
         {
-            PluginManager.SetPlugin(mockHttpPluginWithoutPolly, PluginContract.PlayFab_Transport);
             PluginManager.SetPlugin(mockHttpPluginWithPolly, PluginContract.PlayFab_Transport, "PluginWithPolly");
```

Comment:
This test won't actually test Polly. Since you're providing an instanceName, the clientApi call will just use the default transport plugin.

Reply:
How else do you differentiate the same PluginContract type?

Reply:
Ok i see we just cant have more that one Contract of type PluginContract.PlayFabTransport.